website/public/js/controllers/notificationCtrl.js

```diff
@@ -148,8 +148,9 @@ habitrpg.controller('NotificationCtrl',
     });
 
     // Quest invitation modal
-    $rootScope.$watch('party.quest.key && !party.quest.active && party.quest.members[user._id] == undefined', function(after, before){
-      if (after == before || after != true) return;
+    $rootScope.$watch('user.party.quest.RSVPNeeded == true', function(after, before){
+      if (after != true) return;
+      $rootScope.party.$get();
```

Comment:
This no longer throws an error when `party.quest.key` is added to the watch check.